config-sample-windows.yaml

```diff
@@ -0,0 +1,51 @@
+# Path to a folder containing the GOG db files.
+db_path: C:\Users\my_user\Documents\gog_db
+
+# Valid values are info or debug
+log_level: info
+
+# If set to server, runs as a web service; otherwise runs as a CLI
+mode: server
+
+# The network interface to use when running in server mode
+interface: 0.0.0.0
+
+# The network port to use when running in server mode
+port: 80
+
+# IGDB client setup
+# Instructions for how to obtain a client ID & secret can be found here: https://api-docs.igdb.com/#about
+igdb_client_secret: abcdefghi01234jklmno56789pqrst
+igdb_client_id: 0123uvwxyz4567abcde89012fg34hi
+
+# Path to a file to cache the IGDB setup from
+cache: C:\Users\my_user\Documents\gog-cache.json
+
+# The GOG user ID matching the result of 'select * from Users;' in the DB
+users:
+  12345:
+    username: Bob
+    db: bob-galaxy-2.0.db
+  56789:
+    username: Doug
+    db: doug-galaxy-2.0.db
+metadata:
+  # Each title must match exactly as listed with --all-games; normal YAML quoting rules apply
```

Comment:
I need to update this; the match now happens on the sanitized title so this isn't really true, but it's still a good idea to put it exactly as listed by `--all-games`.